<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">

  <!-- ############## -->
  <!-- Header Content -->
  <!-- ############## -->
  <head>

    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
    <meta name="generator" content="pandoc"/>


    <title>hiPSC-CM Protocol Server</title>
      
    <script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
    <script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
    <script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
    <script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
    <script src="site_libs/navigation-1.1/tabsets.js"></script>

    <!-- Favicons for Apple and Windows users -->
    <link rel="apple-touch-icon-precomposed" sizes="57x57" href="icon/apple-touch-icon-57x57.png" />
    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="icon/apple-touch-icon-114x114.png" />
    <link rel="apple-touch-icon-precomposed" sizes="72x72" href="icon/apple-touch-icon-72x72.png" />
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="icon/apple-touch-icon-144x144.png" />
    <link rel="apple-touch-icon-precomposed" sizes="60x60" href="icon/apple-touch-icon-60x60.png" />
    <link rel="apple-touch-icon-precomposed" sizes="120x120" href="icon/apple-touch-icon-120x120.png" />
    <link rel="apple-touch-icon-precomposed" sizes="76x76" href="icon/apple-touch-icon-76x76.png" />
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="icon/apple-touch-icon-152x152.png" />
    <link rel="icon" type="image/png" href="icon/favicon-196x196.png" sizes="196x196" />
    <link rel="icon" type="image/png" href="icon/favicon-96x96.png" sizes="96x96" />
    <link rel="icon" type="image/png" href="icon/favicon-32x32.png" sizes="32x32" />
    <link rel="icon" type="image/png" href="icon/favicon-16x16.png" sizes="16x16" />
    <link rel="icon" type="image/png" href="icon/favicon-128.png" sizes="128x128" />

    <meta name="application-name" content="&nbsp;"/>
    <meta name="msapplication-TileColor" content="#FFFFFF" />
    <meta name="msapplication-TileImage" content="icon/mstile-144x144.png" />
    <meta name="msapplication-square70x70logo" content="icon/mstile-70x70.png" />
    <meta name="msapplication-square150x150logo" content="icon/mstile-150x150.png" />
    <meta name="msapplication-wide310x150logo" content="icon/mstile-310x150.png" />
    <meta name="msapplication-square310x310logo" content="icon/mstile-310x310.png" />

    <!-- Basic analytics to track views and downloads -->
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-106608402-1', 'auto');
      ga('set', 'anonymizeIp', true);
      ga('send', 'pageview');
    </script>

    <!-- Function that tracks a click on an outbound link in Analytics. From: https://hacks.mozilla.org/2016/01/google-analytics-privacy-and-event-tracking -->
    <script>
    var trackOutboundLink = function(url) {
        // check if the GA object exists and that it has initialized
        if(typeof ga !== 'undefined' && ga.loaded) {
            // if yes, rely on GA to follow link
            ga('send', 'event', 'outbound', 'click', url, {
                'transport': 'beacon',
                'hitCallback': function(){document.location = url;}
            });
        } else {
            // if not, follow link ourselves
            document.location = url;
        }
    }
    </script>

    <style type="text/css">
        h1 {
            font-size: 34px;
        }

        h1.title {
            font-size: 38px;
        }

        h2 {
            font-size: 30px;
        }

        h3 {
            font-size: 24px;
        }

        h4 {
            font-size: 18px;
        }

        h5 {
            font-size: 16px;
        }

        h6 {
            font-size: 12px;
        }

        .table th:not([align]) {
            text-align: left;
        }
    </style>
        
    <link rel="stylesheet" href="sandflat.min.css" type="text/css"/>
    <link rel="stylesheet" href="dt_style.css" type="text/css"/>

  </head>

  <!-- ############ -->
  <!-- Body content -->
  <!-- ############ -->
  <body>

    <style type="text/css">
        .main-container {
            max-width: 940px;
            margin-left: auto;
            margin-right: auto;
        }

        code {
            color: inherit;
            background-color: rgba(0, 0, 0, 0.04);
        }

        img {
            max-width: 100%;
            height: auto;
        }

        .tabbed-pane {
            padding-top: 12px;
        }

        button.code-folding-btn:focus {
            outline: none;
        }
    </style>

    <style type="text/css">
        /* Padding for bootstrap navbar */
        body {
            padding-top: 51px;
            padding-bottom: 40px;
        }

        /* Offset scroll position for anchor links (for fixed navbar) */
        .section h1, 
        .section h2, 
        .section h3, 
        .section h4, 
        .section h5, 
        .section h6 {
            padding-top: 56px;
            margin-top: -56px;
        }
    </style>

<!-- Manages active state of menu based on current page -->
    <script>
      $(document).ready(function () {
        // active menu anchor
        href = window.location.pathname
        href = href.substr(href.lastIndexOf('/') + 1)
        if (href === "")
          href = "index.html";
        var menuAnchor = $('a[href="' + href + '"]');

        // mark it active
        menuAnchor.parent().addClass('active');

        // if it's got a parent navbar menu mark it active as well
        menuAnchor.closest('li.dropdown').addClass('active');
      });
    </script>
    
    <!-- This is the container for the body of the website. This is the primary cause of margins and padding bugs -->
    <div class="container-fluid main-container">

      <!-- Builds tabsets -->
      <script>
        $(document).ready(function () {
          window.buildTabsets("TOC");
        });
      </script>

      <!-- The backbone of the server. Nagivation bar -->
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
        <div class="container">

          <!-- Navbar Header: Includes the brand name and the toggle button for collapsing the navbar -->
          <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <!-- Brand name -->
            <a class="navbar-brand" href="homepage.html">hiPSC-CM Protocol Mining</a>
          </div> <!-- navbar-header -->

          <!-- Navbar links and dropdowns -->
          <div id="navbar" class="navbar-collapse collapse">
            <ul class="nav navbar-nav">
              <li class="dropdown">
                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
                  Study Database<span class="caret"></span>
                </a>
                <ul class="dropdown-menu" role="menu">
                  <li><a href="latest_database.html">Latest Version</a></li>
                  <li><a href="original_database.html">Original Version</a></li>
                </ul> <!-- dropdown-menu -->
              </li> <!-- dropdown -->

              <li><a href="result_plots.html">Enriched Protocols</a></li>

              <li class="dropdown">
                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
                  Methods<span class="caret"></span>
                </a>
                <ul class="dropdown-menu" role="menu">
                  <li><a href="methods_overview.html">Overview</a></li>
                  <li><a href="standard_terms.html">Standardising Terminologies</a></li>
                  <li><a href="pCategory_define.html">Defining Protocol Categories</a></li>
                  <li><a href="fit_dt.html">Configuring Decision Tree</a></li>
                </ul> <!-- dropdown-menu -->
              </li> <!-- dropdown -->
            </ul> <!-- nav navbar-nav -->

            <ul class="nav navbar-nav navbar-right">
              <li class="dropdown">
                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
                  About<span class="caret"></span>
                </a>
                <ul class="dropdown-menu" role="menu">
                  <li><a href="credits.html">Credits & Citations</a></li>
                  <li><a href="https://github.com/phycochow/palpantlab_hiPSC_database">Server Code</a></li>
                  <li><a href="https://imb.uq.edu.au/research-groups/palpant">Palpant Group</a></li>
                </ul> <!-- dropdown-menu -->
              </li> <!-- dropdown -->
            </ul> <!-- nav navbar-nav navbar-right -->

          </div> <!-- navbar-collapse -->

        </div> <!-- container -->
      </div> <!-- navbar navbar-default navbar-fixed-top-->

      <!-- Title -->
      <div class="fluid-row" id="header">
        <h2 class="title toc-ignore">Differentiation and Maturation Protocols Influences the Functionaility of Stem Cell Derived Cardiomyocytes</h1>
      </div>

      <div class="well">
        <p>
          We performed data mining on 300 hiPSC-CM studies (Ewoldt et al., 2024 manuscript) to find common themes that influence physiological performance and establish a clear knowledge gap in how variation in protocol parameters, like genetic background of hiPSCs, affect maturation and downstream hiPSC-CM function. It also points at the limited number and diversity within in-vitro datasets. And despite the transcriptomic insights of genome-wide CRISPR screens (Padmanabhan et al., 2024), hiPSC-CMs do not fully recapitulate complex cell and organ-level physiology that can be causal of traits (Cao et al., 2023), which shifts this project’s attention to alternative data resources. The <a href="https://imb.uq.edu.au/research-groups/palpant">Palpant Lab</a> has been working to make some basic analyses of that data freely available; you can read more about the overall effort <a href="http://www.nealelab.is/uk-biobank">here</a>.
        </p>
        
        <p><br></p> <!-- line break -->

        <p>
          Building on the initial genome-wide association analyses, we’ve now estimated the heritability of each of those traits and disorders, over 4,000 as of this writing. As with the primary GWAS analyses, we’re making the LD Score Regression SNP-heritability results available for <a href="h2_browser.html">browsing</a> and bulk <a href="downloads.html">download</a>.
        </p>
        
        <p>
          If you’re interested in additional traits, additional ancestry groups, and/or other heritability estimation methods, SNP heritability estimates are also now available from the <a href="https://pan.ukbb.broadinstitute.org/docs/heritability">Pan-UK Biobank analysis</a>.
        </p>
        
        <p><br></p>
        <p>
          <a href="h2_browser.html" class="btn btn-primary btn-lg">Browse <span class="math inline">\(h^2\)</span></a>
          <a href="viz_h2.html" class="btn btn-primary btn-lg">Explore Plots</a>
          <a href="downloads.html" class="btn btn-primary btn-lg">Download Results</a>
        </p>
        
        <p><br></p>
        <p>You can learn more about the Round 2 Neale Lab GWAS heritability analysis in this series of blog posts:</p>
        <ul>
          <li><a href="http://www.nealelab.is/blog/2017/9/13/heritability-101-what-is-heritability">Heritability 101: What is “heritability”?</a></li>
          <li><a href="http://www.nealelab.is/blog/2017/9/13/heritability-201-types-of-heritability-and-how-we-estimate-it">Heritability 201: Types of heritability and how we estimate it</a></li>
          <li><a href="http://www.nealelab.is/blog/2017/9/20/insights-from-estimates-of-snp-heritability-for-2000-traits-and-disorders-in-uk-biobank">Insights from estimates of SNP-heritability for &gt;2,000 traits and disorders in UK Biobank</a></li>
        </ul>
        
        <p>More technical descriptions are also available:</p>
        <ul>
          <li><a href="http://www.nealelab.is/blog/2017/9/14/heritability-501-ldsr-based-h2-in-ukbb-for-the-technically-minded">Heritability 501: LDSR-based <span class="math inline">\(h^2\)</span> in UKBB for the technically-minded</a></li>
          <li><a href="https://github.com/Nealelab/UKBB_ldsc_scripts">UKBB_ldsc_scripts GitHub repository</a></li>
        </ul>
        
        <p>
          We view these results as preliminary and subject to change. They rely on uniform automated GWAS of thousands of variables, each of which can potentially be refined by subject matter experts. You can <a href="viz_R1vR2.html">view the change between our Round 1 and Round 2 results here</a>. But we hope you’ll agree that the current results still provide an interesting and useful look at the genetics of the many diverse traits and disorders studied in the UK Biobank, and are potentially informative for highlighting phenotypes meriting further follow-up.
        </p>
        
        <p>
          If you want more information, please <strong>contact us at: <a href="mailto:nealelab.ukb@gmail.com">nealelab.ukb@gmail.com</a>.</strong>
        </p>
      </div>


    </div> <!-- container-fluid main-container -->
  </body>
</html>

